programs/clearing_house/src/instructions/admin.rs

```diff
@@ -712,7 +713,19 @@ pub fn handle_settle_expired_market_pools_to_revenue_pool(
     )?;
 
     validate!(
-        now > perp_market.expiry_ts + TWENTY_FOUR_HOUR,
+        state.settlement_duration != 0,
+        ErrorCode::DefaultError,
+        "invalid state.settlement_duration (is 0)"
+    )?;
+
+    let escrow_period_before_transfer = if state.settlement_duration > 1 {
+        TWENTY_FOUR_HOUR * cast_to_i64(state.settlement_duration - 1)?
+    } else {
+        cast_to_i64(state.settlement_duration)?
```

Comment:
why do you make settlement a multiple of 24 hours or just make it 1? for testing? could add a comment for clarity.